docs/content/en/docs/tutorials/ci_cd.md

```diff
@@ -0,0 +1,286 @@
+
+**Step 5: The skaffold.yaml file**
+
+Depending on the way you would like to build, deploy, your skaffold yaml file will actually change and it's quite easy to set up one. It is recommended that you set up skaffold before you setup your CI pipeline since the CI pipeline can just call skaffold to do all the builds and deployments and you can do the same locally as well.
+
+A sample skaffold file can look something like this:
+
+
+```
+apiVersion: skaffold/v2beta8
+kind: Config
+profiles:
+  - name: my-profile-name
+    build:
+      artifacts:
+        - image: asia.gcr.io/my-project/my-image
+          docker:
+            buildArgs:
+              NPM_REGISTRY: '{{.NPM_REGISTRY}}'
+              NPM_TOKEN: '{{.NPM_TOKEN}}'
+    deploy:
+      helm:
+        releases:
+          - name: my-release
+            namespace: default
+            useHelmSecrets: true
+            chartPath: ./charts/my-chart
+            valuesFiles:
+              - ./deployment/dev/values.yaml
+              - ./deployment/dev/secrets.yaml
+            skipBuildDependencies: true
+            artifactOverrides:
+              image: asia.gcr.io/my-project/my-image
+            imageStrategy:
+              helm: {}
+        flags:
+          upgrade:
+            - --install
+```
+
+
+Here, we are
+
+
+
+*   **Creating a profile** - You can set up multiple profiles using skaffold and deploy them separately if you want using **skaffold run -p &lt;profilename>**
+  
+*   **Specifying the artifacts to build from** - In this case, we are asking skaffold to build from a Dockerfile, passing its args and also specifying the destination where the image needs to be pushed. The values can also come from the env (NPM_REGISTRY and NPM_TOKEN in this example)
+
+*   **Specifying the deployment strategy** - In this case, helm is specified with the location of the chart, all the configuration as documented in the yaml reference and also the values to be passed to it. We are also using the helm secrets plugin above.
+
+
+If you want to deploy via kubectl, your skaffold file can look something like this:
+
+
+```
+apiVersion: skaffold/v2beta8
+kind: Config
+profiles:
+    - name: dev-svc
+      build:
+          artifacts:
+              - image: asia.gcr.io/my-project/my-image
+                docker:
+                    buildArgs:
+                        NPM_REGISTRY: '{{.NPM_REGISTRY}}'
+                        NPM_TOKEN: '{{.NPM_TOKEN}}'
+      deploy:
+          statusCheckDeadlineSeconds: 600
+          kubectl:
+              manifests:
+                  - k8/dev/*.yml
+```
+
+
+**Please refer the yaml reference for all that you can specify in skaffold.yaml file**
+
+
+## **Step 6: Development & Testing locally**
```

Comment:
The labels for steps 1-5 use emphasis, but steps 6-9 use emphasis and are also headers. This styling should be consistent throughout.